### PR TITLE
Test properly that if there is a saxon script, it is the EXPath one.

### PR DESCRIPTION
**Pull request by fgeorges**
See #112 and #121.

### CODE REVIEW

File: bin/xspec.sh [hunk 1]
```diff
@@ -59,7 +59,7 @@ die() {
 # configured it, so there is no point to duplicate the logic here.
 # Just use it.
 
-if which saxon > /dev/null 2>&1; then
+if which saxon > /dev/null 2>&1 && saxon --help > /dev/null 2>&1 && saxon --help | grep "EXPath Packaging" > /dev/null 2>&1; then
```
Comment (cirulls) [inline bin/xspec.sh]: Thanks for this. I suggest to keep only the third condition to test that the saxon script exists and is the EXPath one:
```shell
if saxon --help | grep "EXPath Packaging" > /dev/null 2>&1; then
```
If the saxon script is not present the command ```saxon --help``` returns an error code which evaluates to ```false```. If a saxon script is present but it's not the EXPath packaging one, the command ```grep "EXPath Packaging"``` returns an empty result which evaluates to ```false```. Therefore I think that the other conditions ```which saxon > /dev/null 2>&1 && saxon --help > /dev/null``` are not necessary.

Comment (fgeorges) [inline bin/xspec.sh]: Whatever, you're in charge.

**Comment by cirulls** [comment]
Please see #124.

**Comment by cirulls** [comment]
I close this pull request as it is integrated in #124.

**Pull request closed without merging**